### ISSUE TITLE
Wie hochgeladene Pdf Datei aus den Platzregeln wieder entfernen

### ISSUE THREAD

**Issue by tennstar1** [tkrebs/ep3-bs#466]
Hallo,
ich wollte jetzt während Corona eine PDF Datei in den Platzregeln beim buchen anzeigen und bestätigen lassen.
Die Datei wurde hochgeladen und man wird auch aufgefordert sie zu bestätigen, sie wird aber nicht angezeigt.
Stattdessen kommt ein "Error 500 - Internal server error" Fehler.
Ich würde diesen Eintrag der  Datei gerne wieder entfernen weiß aber nicht wie!
Die Datei selbst habe ich aus dem Upload Ordner entfernt!

Weiß bitte jemand wo ich den Eintrag löschen kann, es hilft auch nichts die Regeln zu ändern und alles zu löschen, der Eintrag der Datei bleibt.
Gruß tennstar

**Comment by luke8800gts** [comment]
Ja, ich habe mich auch schon gefragt, wie das entfernen dieser datei funktioniert. (siehe #307)
Nicht haben die Kenntnisse um das über die Datenbank zu machen.

Bitte das Enfernen implementieren bzw. die Möglichkeit dazu besser anzeigen, falls schon vorhanden. Danke.

**Comment by tennstar1** [comment]
Hallo luke8800gts,
vielen , vielen Dank für den Hinweis! 👍👍
Such jetzt schon Tage, aber nun ist alles gut!
